install/servlet/docs/index.html

```diff
@@ -64,10 +72,10 @@ <h2>Guide to Jakarta EE Servlet TCK 5.0 Documentation</h2>
       </li>
     </ul>
     <h2>JT Harness Documentation</h2>
-    <p>The online version of the JT Harness version 5.0 documentation is
+    <p>The online version of the JT Harness version 6.0 documentation is
       available <a href="https://wiki.openjdk.java.net/display/CodeTools/Documentation">here</a>.</p>
     <hr>
-    <p><small><em>Copyright � 2017, 2021 Oracle and/or its affiliates. All
+    <p><small><em>Copyright � 2018, 2022 Oracle and/or its affiliates. All
```

Comment:
2017, 2022

Reply:
I aligned that text with the license header which I assume is canonical